python/scripts/worker-report.py

```diff
@@ -1,97 +0,0 @@
-#!/usr/local/bin/python3
-
-"""
-Populate relations.
-Examples:
-	worker populate pipeline.pre.ExtractSpikes
-"""
-import os
-import random
-from argparse import ArgumentParser
-from datetime import datetime
-from importlib import import_module
-from slacker import Slacker
-import time
-import inspect
-import sys
-import pipeline
-import datajoint as dj
-
-addr = ['Despicable masters', 'Evil overlords', 'Dear Villain Scientists', 'Ladies and Gentlemen',
-        'Fellow Slackers', '"Real" people', 'Dear Biomass', 'Datajoyous greetings']
-
-
-def post_report(comment, report, token=None, channel='#bot_planet'):
-    if token is not None:
-        slack = Slacker(token)
-        with open('report.txt', 'w') as fid:
-            fid.write(report)
-        slack.files.upload('report.txt', filetype='text', title='Minion report', initial_comment=comment,
-                           channels=channel)
-    else:
-        print(report)
-
-
-def progress_bar(k, n, barlen=20):
```

Comment:
That might be nice for `populate` but the worker just works entirely in the background.